src/dom-utils/listScrollParents.js

```diff
@@ -11,8 +11,11 @@ export default function listScrollParents(
 ): Array<Element | Window> {
   const scrollParent = getScrollParent(element);
   const isBody = getNodeName(scrollParent) === 'body';
-  const target = isBody ? getWindow(scrollParent) : scrollParent;
-  const updatedList = list.concat(target);
+  const win = getWindow(scrollParent);
+  const target = isBody ? win : scrollParent;
+  const updatedList = list.concat(
+    isBody && win.visualViewport ? [target, win.visualViewport] : target
+  );
```

Comment:
```suggestion
  const updatedList = list.concat(target, win.visualViewport || []);
```

This may be a bit more readable.